core/arch/arm/kernel/abort.c

```diff
@@ -157,6 +211,7 @@ void abort_print_error(struct abort_info *ai __maybe_unused)
 	     read_mpidr_el1(), (uint32_t)ai->regs->spsr);
 #endif /*ARM64*/
 #endif /*TRACE_LEVEL >= TRACE_DEBUG*/
+	print_stack_unwind(ai);
```

Comment:
The function is only used from EMSG macros so it could be unused depending on config flags.